run_data_exploration.py

```diff
@@ -111,12 +165,84 @@ def get_csv(csv):
 for i in range(n_plot_nrs):
     start = n_plot_types*i
     stop = n_plot_types*(i+1)
-    # TODO: change width variable, width could be dependent on screen resoluation
+    # TODO: change width variable, dependent on screen resolution
     # TODO: improve caption
-    st.image([f'./output/{output_plots}/{plt}' for plt in list_plots[start:stop]],
+    st.image([f'./output/{output_plots}/{plt}' for plt in list_plots[
+                                                           start:stop]],
              width=400, caption=[plt[:-4] for plt in list_plots[start:stop]])
 
 
+st.header('LR results')
+# get LR results, same as exp.results
+latest_lr_csv = sorted([f for f in (os.listdir('output')) if f.endswith(
+    'lr_results.csv')])[-1]
+
+if len(latest_lr_csv) == 0 or latest_exp_csv[:19] != latest_lr_csv[:19]:
+    st.markdown('No LR results available.')
+else:
+    df_models = get_csv(latest_lr_csv)
+    df_models['pair_id'] = df_models.apply(lambda row: f'{row.pair_id[:-2]}',
+                                           axis=1)
+    df_models['model'] = df_models.apply(lambda row: f'{row.scorers}_{row.calibrators}',
+                                           axis=1)
+    set_models = list(set(df_models['model']))
+
+    df_enfsi = get_enfsi_lrs()
+    df_lrs = df_enfsi
+
+    for model in set_models:
+        df_lrs[f'{model}'] = np.nan
+
+    for index, row in df_models.iterrows():
+        df_lrs.loc[(df_lrs['pair_id'] == row['pair_id']), [row['model']]] = \
+            np.log10(row['LR'])
+
+    df_lrs = df_lrs[df_lrs[set_models[0]].notnull()]
+    df_lrs_long = pd.melt(df_lrs, id_vars='pair_id', value_vars=list(df_lrs)[
+                                                               3:],
+                          var_name='model', value_name='LR')
+
+    df_lrs_long.loc[df_lrs_long['model'].isin(range(0, 100)), 'model'] = \
+        'expert'
+
+    set_calibrators = list(set(df_models['calibrators']))
+    set_scorers = list(set(df_models['scorers']))
+
+    st.subheader('General information')
+    st.markdown(f'latest LR csv: {latest_lr_csv}')
+    st.markdown(f'scorers: {set_scorers}')
+    st.markdown(f'calibrators: {set_calibrators}')
+
```

Comment:
kunnen we hier ook wat summarising stats laten zien? Bijv de cllr van de experts (met spreiding?) en de systemen? en per enfsi jaar? en evt de accuracy (dan is kalibratie van experts minder belangrijk)

Reply:
cllrs samen toegevoegd.